scripts/nixfmt-mergetool

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 Jan Malakhovski <oxij@oxij.org>
+#
+# This file can be redistributed under the terms of Unlicense
+# <https://unlicense.org> license.
```

Comment:
The only worry I have is the re-introduction of separately licensed files, which we moved away from in the past: https://github.com/NixOS/nixfmt/pull/171. Would you be okay with you to re-license it under the MPL-2.0 license and assign copyright to the generic "Nixfmt contributors" as defined in https://github.com/NixOS/nixfmt/blob/e825e956ae967ee24aa502a90099956d1f8bc1ed/.reuse/dep5#L3-L6?

```suggestion
```